02_bubble/lexual/sort.go

```diff
@@ -0,0 +1,47 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+func bubble(s []int) []int {
+	t := make([]int, len(s))
+	copy(t, s)
+
+	n := len(s)
+	for {
```

Comment:
avoid using forever loops if possible;
using 
```for n > 1```
 will have the same outcome and then you can remove 
```
if n <= 1 {
    break
}
```